joblib/memory.py

```diff
@@ -611,8 +627,10 @@ def __getstate__(self):
     # ------------------------------------------------------------------------
 
     def _get_argument_hash(self, *args, **kwargs):
-        return hashing.hash(filter_args(self.func, self.ignore, args, kwargs),
-                            coerce_mmap=(self.mmap_mode is not None))
+        return self.hash_func(
+            filter_args(
+                self.func, self.ignore, args, kwargs), coerce_mmap=(
+                self.mmap_mode is not None))
```

Comment:
Not so easy to read this way, I propose:
```suggestion
        return self.hash_func(
            filter_args(self.func, self.ignore, args, kwargs),
            coerce_mmap=self.mmap_mode is not None
        )
```